home/tests.py

```diff
@@ -1,3 +1,34 @@
-# from django.test import TestCase
+from django.contrib.auth.models import User
+from home.models import Profile, Question, Answer
+import pytest
 
-# Create your tests here.
+
+@pytest.mark.django_db
+class TestModels:
```

Comment:
You don;t need to put tests in classes when using pytest - if you do, its better to give the classes some speficig names that indicate which tests do they group together - and it doesn;t ususally make sense t group all the model tests together in the same class...